test/Microsoft.ML.CpuMath.PerformanceTests/Microsoft.ML.CpuMath.PerformanceTests.csproj

```diff
@@ -3,7 +3,6 @@
     <OutputType>Exe</OutputType>
     <LangVersion>7.2</LangVersion>
     <StartupObject>Microsoft.ML.CpuMath.PerformanceTests.Program</StartupObject>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
```

Comment:
This perf tests reference CpuMath and should target netcoreapp3.0 to get the new implementation, right?

Reply:
Oops. Yep this was too much slashing.